libs/utils/energy.py

```diff
@@ -118,6 +121,58 @@ def reset(self):
     def report(self, out_dir):
         raise NotImplementedError('Missing implementation')
 
+class Monsoon(EnergyMeter):
+    def print_log(self, errmsg, exception=0):
+	print errmsg
+	self._log.error(errmsg)
+	if exception == 1:
+		raise(Exception(errmsg))
+
+    def __init__(self, target, conf=None):
+	super(Monsoon, self).__init__(target, '')
+
+	# Read config
+	# { 'config': { 'device_entry' : '/dev/ttyACM1', 'voltage' : 4.3 } }
+	try:
+		device_entry = conf['config']['device_entry']
```

Comment:
You can avoid the try/catch by using:

```python
self.device = conf['config'].get('device', None)
self.voltage = conf['config'].get('voltage', 4.3)
```

I would also prefer to `s/device_entry/device/`